agent/util-scripts/gold/pbench-metadata-log/test-03.txt

```diff
@@ -68,33 +79,31 @@ turbostat: --interval=3
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/proc-vmstat:foobar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/sar:foo
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/turbostat:bar
 /var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -o StrictHostKeyChecking=no -n dhcp31-174 hostname -s
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/scp -r perf104:/var/tmp/pbench-test-utils/pbench/tools-default /var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/iostat:foo
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/mpstat:bar
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/perf:baz
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/pidstat:hunoz
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/proc-interrupts:hukerz
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/proc-vmstat:foobar
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/sar:foo
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/tools-default/iostat:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/perf104/turbostat:bar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/scp -r perf104:/var/tmp/pbench-test-utils/pbench/tools-default /var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/iostat:foo
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/mpstat:bar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/perf:baz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/pidstat:hunoz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/proc-interrupts:hukerz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/proc-vmstat:foobar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/sar:foo
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/turbostat:bar
```

Comment:
Ditto here as well for `.../perf104/tools-default/iostat` ...

Reply:
Ditto.